### ISSUE TITLE
Insert new line at the end of file

### ISSUE THREAD

**Issue by watagit** [scaffdog/scaffdog#437]
Hi. I am a regular user of this tool. Thank you all for your work.
This time I have a question.
I want to insert one blank line at the end of an auto-generated file.
I put the blank line in the markdown, but it does not work as expected.
For example, writing a markdown like the one below will not insert a blank line at the end of the auto-generated file.
```
import type { VFC } from "react";

export const Profile: VFC = () => {
  return <main>Profile</main>;
};
# newline
```

My project requires a blank line at the end of the file, so I would like to insert it somehow.
Is there any way to solve this?

## Your Environment

- OS: MacOS Monterey 12.5
- scaffdog version: 2.0.3

**Comment by watagit** [comment]
We found that we could insert blank lines this way. Is there a smarter way?
```
import type { VFC } from "react";

export const {{ inputs.name }}: VFC = () => {
  return (
    <main>{{ inputs.name }}</main>
  );
};
{{ null }}
```

**Comment by wadackel** [comment]
@watagit 

Hi, thank you for using scaffdog!

The current template engine implements trimming the beginning and end of the file.
So adding an empty line requires the workaround you suggested :cry:

**Comment by watagit** [comment]
@wadackel 

Thank you for your response.
I see, so that is the specification.

Just out of curiosity, what is the motivation for trimming the beginning and end of the file?
Many codes should expect to insert a single blank line at the end, and I personally have felt that there is no need to trim.
If you have any past discussions that might be helpful, I would appreciate you sharing them.

**Comment by wadackel** [comment]
> Just out of curiosity, what is the motivation for trimming the beginning and end of the file?

I did not have a strong motivation. But, when formatting code using prettier, etc., the first and last blank lines are trimmed. So, I've never felt inconvenienced by being trimmed :dog:

But, Since the philosophy of scaffdog is to output the given input in a straightforward manner, we would like to modify it with a policy of not performing unnecessary trimming.

**Comment by wadackel** [comment]
@watagit 

Released as [v2.1.1](https://www.npmjs.com/package/scaffdog/v/2.1.1) :rocket:
Try this version when you have time.

**Comment by watagit** [comment]
@wadackel 

Thank you very much!!

> Since the philosophy of scaffdog is to output the given input in a straightforward manner

The philosophy is so great. I like this tool more now!

I will report back to you if I hear anything else.
Thank you very much for your cooperation.
